data/json/items/comestibles/egg.json

```diff
@@ -742,21 +742,22 @@
   {
     "type": "COMESTIBLE",
     "id": "rehydrated_eggs",
-    "copy-from": "egg_bird",
-    "name": { "str": "rehydrated egg" },
+    "name": { "str_sp": "liquid eggs" },
```

Comment:
why overriding rehydrated eggs here? Just make a new item, it's weird that mixing eggs makes rehydrated eggs when they should be made from egg powder

Reply:
I once heard from food factories that rehydrated eggs contain all the nutrients of eggs except the shell.  Rehydrated eggs are (egg powder + water) = all the nutrients of the egg liquid, which is the same as the nutrition obtained by directly crushing an egg.  Some time ago, I added the item "rehydrated eggs" to allow the protagonist to mix different kinds of eggs.  But I found that this didn't quite serve the purpose – because making whole egg flour itself requires the same kind of eggs, so this change was made.